price_service/server/package.json

```diff
@@ -15,24 +15,24 @@
   "author": "",
   "license": "Apache-2.0",
   "devDependencies": {
-    "@types/jest": "^27.5.0",
+    "@types/jest": "^29.4.0",
     "@types/long": "^4.0.1",
     "@types/node": "^16.6.1",
     "@types/node-fetch": "^2.6.2",
     "@types/supertest": "^2.0.12",
-    "jest": "^28.0.3",
+    "jest": "^29.4.0",
```

Comment:
Strangely the older jest was crashing on running the tests due to an uninstalled dependency of itself, bumping the version solved it. (probably it was moving lock file issue)